classes/PodsInit.php

```diff
@@ -1875,6 +1931,8 @@ public function run() {
 		add_action( 'wp_enqueue_scripts', array( $this, 'register_assets' ), 15 );
 		add_action( 'admin_enqueue_scripts', array( $this, 'register_assets' ), 15 );
 		add_action( 'login_enqueue_scripts', array( $this, 'register_assets' ), 15 );
+		// Elementor Page Builder.
+		add_action( 'elementor/editor/before_enqueue_scripts', array( $this, 'register_assets' ), 15 );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: c84c972a976cba46aa6bdc38a86b3b85</sub>